setup.py

```diff
@@ -19,7 +19,7 @@
         "service-identity>=14.0.0",
         "treq>=15.0.0",
         "pyOpenSSL>=0.15.1",
-        "txaio==2.1.0",
+        "txaio==2.2.0",
```

Comment:
I couldn't get it to work on OSX with 2.1.0 hence the bump.  I can try both again today if that's the case